platform/wayland/cog-view-wl.c

```diff
@@ -87,12 +105,20 @@ cog_wl_view_init(CogWlView *self)
     self->should_update_opaque_region = true;
 
     self->scale_factor = 1;
+    self->should_update_opaque_region = true;
     self->image = NULL;
     self->frame_callback = NULL;
     self->platform = COG_WL_PLATFORM(cog_platform_get_default());
 
+    // Only one view is managed by the Wayland Platform nowadays.
+    self->platform->view = self;
+
     wl_list_init(&self->shm_buffer_list);
 
+    g_signal_connect(self, "mouse-target-changed", G_CALLBACK(on_mouse_target_changed), NULL);
+#if COG_HAVE_LIBPORTAL
+    g_signal_connect(self, "run-file-chooser", G_CALLBACK(on_run_file_chooser), NULL);
+#endif /* COG_HAVE_LIBPORºTAL */
```

Comment:
It looks like your cat got onto the keyboard 🐈:

```suggestion
#endif /* COG_HAVE_LIBPORTAL */
```

Reply:
fixed.